etl/snapshot.py

```diff
@@ -228,6 +228,8 @@ def read(self, *args, **kwargs) -> Table:
             return self.read_json(*args, **kwargs)
         elif self.metadata.file_extension == "dta":
             return self.read_stata(*args, **kwargs)
+        elif self.metadata.file_extension == "rds":
+            return self.read_rds(*args, **kwargs)
```

Comment:
For completeness, you could also add here `read_rda`, and define the method below (given that the function already exists in module `processing`).